atomic_reactor/plugins/exit_remove_worker_metadata.py

```diff
@@ -0,0 +1,46 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from atomic_reactor.plugin import ExitPlugin
+from atomic_reactor.plugins.build_orchestrate_build import get_worker_build_info
+from atomic_reactor.constants import PLUGIN_REMOVE_WORKER_METADATA_KEY
+
+
+class RemoveWorkerMetadataPlugin(ExitPlugin):
+    """
+    Remove worker metadata for each platform.
+    """
+
+    key = PLUGIN_REMOVE_WORKER_METADATA_KEY
+
+    def run(self):
```

Comment:
Most of the code in this plugin is a direct duplication of `fetch_worker_metadata`.
Let's abstract it.

Reply:
done